space/space.go

```diff
@@ -54,6 +55,7 @@ type Repository interface {
 	Load(ctx context.Context, ID satoriuuid.UUID) (*Space, error)
 	Delete(ctx context.Context, ID satoriuuid.UUID) error
 	List(ctx context.Context, start *int, length *int) ([]*Space, uint64, error)
```

Comment:
Sure, but which service/controller would have the need to paginate through a list of 1million spaces? If so it sounds like they are doing something wrong. :)